### PR TITLE
[WIP] Ember 2.x upgrades + fixes

### PR DESCRIPTION
**Pull request by noopkat**
Hello! 👋 

Thank you for writing this add-on! We're looking to use this in both our dev and CI flow here at work, and are deprecating [ember-a11y-testing](https://github.com/trentmwillis/ember-a11y-testing) as advised.

**I'm opening this PR as an early WIP / RFC.**

I thought now would be a good opportunity to update the dependencies on this add-on so we can have clean slate for regular maintenance going forward (which I'm happy to volunteer with).

I also rolled in the fragment fix from #5 in as I was running into the same error with fragments in our stack.

Travis config has added newer Node versions added to ensure everything is passing there.

Lastly, there was a bug we were experiencing to do with red border highlighting the violating ember component on the page. The DOM selector string that axe-core returns as the offending element includes the Ember component's id, eg. `"#ember727 > .bundle__see-more-content > .bundle__see-more-label"`. Using the component's `$.()` method makes it impossible to include itself while hunting for the exact DOM element to highlight. Let me know if I'm missing something obvious though!

There's a performance hit for the fix I implemented, as I wasn't as comfortable with something more fragile such as regex'ing out the component's id from the axe-core `node.target` selector string.
### RFC
- I'm new to Ember add-on architecture. Am I doing this right?
- Will these changes cause any backwards compatibility issues with Ember pre 2.x?
- Should I add multiple Ember versions to the Ember Try config?
### todo
- [ ] get the styles working again for the red border on violations in the UI (dev flow)
- [ ] Remove the version bump.
- [ ] Add back in the keywords.
- [ ] Add ember-lts to the Ember Try config.
- [ ] Include ember-lts and 1.13 into the `travis.yml` file.

### CODE REVIEW

**Comment by noopkat** [comment]
I'll set up an Ember 1.12 / 1.13 app when I get a chance to test this out with, or update the Ember Try config if that's even easier.

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "ember-axe",
-  "version": "0.0.2",
+  "version": "1.0.0",
```
Comment (noopkat) [inline package.json:3]: Happy to change this to a patch level, was being cautious more than anything

File: tests/dummy/app/index.html [hunk 2]
```diff
@@ -7,19 +7,19 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    {{content-for 'head'}}
+    {{content-for "head"}}
```
Comment (noopkat) [inline tests/dummy/app/index.html:10]: The add-on generator caused this pedantry 😁

File: config/ember-try.js [hunk 3]
```diff
@@ -1,34 +1,54 @@
...
   scenarios: [
     {
       name: 'default',
-      dependencies: { }
+      bower: {
+        dependencies: { }
+      }
+    },
+    {
+      name: 'ember-1.13',
```
Comment (ghost) [inline config/ember-try.js:11]: this is great

File: package.json [hunk 4]
```diff
@@ -21,34 +21,35 @@
...
+    "loader.js": "^4.0.1",
+    "ember-sinon": "*"
   },
   "keywords": [
-    "ember-addon",
-    "accessibility",
-    "ember",
-    "a11y",
-    "testing"
+    "ember-addon"
```
Comment (nathanhammond) [inline package.json:47]: Can we add the rest of the keywords back in?

File: package.json (hunk 1 above)
Comment (nathanhammond) [inline package.json:3]: Should be fine as a patch level, but we generally bump the version separate from every other change. Let's not make this change in this PR. 😄

File: config/ember-try.js (hunk 3 above)
Comment (nathanhammond) [inline config/ember-try.js:11]: To make this work you'll need to add a line in `travis.yml`.

**Comment by nathanhammond** [comment]
I added more to the to-do list in the initial post. Once those are all set I can re-review and land. This looks great!

> I'm new to Ember add-on architecture. Am I doing this right?

Yep! Looks perfect.

> Will these changes cause any backwards compatibility issues with Ember pre 2.x?

Nope, the addon will be built with whatever version of ember-cli, but the code itself should be compatible across numerous Ember versions. 

> Should I add multiple Ember versions to the Ember Try config?

And the Ember Try config you've set up will make sure we remain compatible with the versions we wish to support. 😄

File: config/ember-try.js (hunk 3 above)
Comment (noopkat) [inline config/ember-try.js:11]: @nathanhammond will do, thanks!

File: package.json (hunk 1 above)
Comment (noopkat) [inline package.json:3]: @nathanhammond 👍

File: package.json (hunk 4 above)
Comment (noopkat) [inline package.json:47]: gah, thought I fixed this hehe

**Comment by nathanhammond** [comment]
@noopkat Can you reopen this PR against [ember-a11y-testing](https://github.com/trentmwillis/ember-a11y-testing)? We migrated this addon into that repository. After normal updates this PR should still merge. :)

**Pull request closed without merging**